rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -522,6 +523,7 @@ impl Controller {
         });
         self.refresh_system_tray_menu()?;
 
+        ran_before::set().await?;
```

Comment:
We still hide the Welcome window as soon as you click "Sign in" but the window is only permanently disabled on the first successful (ignoring IPC async stuff) connlib start

Reply:
Hm yeah, I think this works. It could make sense if we add sign in instructions to it, too. I need to think if this change is applicable to the Apple clients.

Reply:
Especially with #5014, I wonder how many first-time users will not even see the tray icon. That's part of why I wanted the window to live a little longer